src/app/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 find_package(${QT_PKG}QuickCompiler QUIET)
 
+if(NOT Qt5QuickCompiler_FOUND
+   OR CMAKE_BUILD_TYPE MATCHES Debug
+   OR CMAKE_BUILD_TYPE MATCHES RelWithDebInfo)
+  set(RESOURCES ${CMAKE_SOURCE_DIR}/src/qml/qml.qrc)
+endif()
```

Comment:
This looks unrelated

Reply:
Right, I need to remove this commit. We still need to find a way for QField to compile properly on linux without vcpkg.